public/javascripts/SVLabel/doc/PointCloud.html

```diff
@@ -583,7 +583,11 @@ <h4 class="name" id="~search"><span class="type-signature">(inner) </span>search
 
 
 <div class="description">
-    Given the coordinate x, y (and z), return index of the point cloud data.
To further calculate the x- and y-coordinates, do as follows:

ix = idx / 3 % w
iy = (idx / 3 - ix) / w
+    Given the coordinate x, y (and z), return index of the point cloud data.
```

Comment:
What happened with all these doc files? I understand how the ones with action stack references were changed, but what happened with the line breaks in these ones?